### ISSUE TITLE
Bower Package Management Document errors on VS Reopen

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#10105]
## Details about Problem

NuGet product used: VS UI
VS version (if appropriate): Version 16.8.0 Preview 3.1
Worked before? If so, with which NuGet version: Worked in 15.x, Broken in at least 16.7.4 (not tested earlier)

## Detailed repro steps so we can see the same problem
**Note: On Preview VS, I did not have Bower package versions populated in the dropdown. On Experimental VS, I did have versions populated.**

1. Create a bower-incompatible project, such as .NET Standard 2.0, then Open NuGet Package Manager.

2. Find a package that recommends using the Bower Package Manager, eg, jQuery.
![image](https://user-images.githubusercontent.com/49205731/94961820-6d129400-04c3-11eb-8901-95187a320869.png)

3. Select the Bower link to open the Bower Package Manager window.

4. Close Visual Studio.

5. Reopen Visual Studio.

6. An error is shown
![image](https://user-images.githubusercontent.com/49205731/94962041-cf6b9480-04c3-11eb-9b86-36e821150cd3.png)

## Other suggested things
Relevant: https://github.com/NuGet/Home/issues/6623

**Comment by donnie-msft** [comment]
Note I have been working with InfiniteScrollList for dev and using my Experimental VS. 
Here's the VS Crash:
![image](https://user-images.githubusercontent.com/49205731/94964648-152a5c00-04c8-11eb-90d3-ffd4f7f85202.png)


> System.ArgumentOutOfRangeException
>   HResult=0x80131502
>   Message=Specified index is out of range or child at index is null. Do not call this method if VisualChildrenCount returns zero, indicating that the Visual has no children.
> Parameter name: index
> Actual value was 0.
>   Source=PresentationFramework
>   StackTrace:
>    at System.Windows.FrameworkElement.GetVisualChild(Int32 index)
>    at System.Windows.Media.Visual.InternalGet2DOr3DVisualChild(Int32 index)
>    at System.Windows.Media.VisualTreeHelper.GetChild(DependencyObject reference, Int32 childIndex)
>    at NuGet.PackageManagement.UI.InfiniteScrollList.List_Loaded(Object sender, RoutedEventArgs e) in C:\NuGet.Client\src\NuGet.Clients\NuGet.PackageManagement.UI\Xamls\InfiniteScrollList.xaml.cs:line 694
>    at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
>    at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
>    at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
>    at System.Windows.UIElement.RaiseEvent(RoutedEventArgs e)
>    at System.Windows.BroadcastEventHelper.BroadcastEvent(DependencyObject root, RoutedEvent routedEvent)
>    at System.Windows.BroadcastEventHelper.BroadcastLoadedEvent(Object root)
>    at MS.Internal.LoadedOrUnloadedOperation.DoWork()
>    at System.Windows.Media.MediaContext.FireLoadedPendingCallbacks()
>    at System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
>    at System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)
>    at System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)
>    at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
>    at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
>    at System.Windows.Threading.DispatcherOperation.InvokeImpl()
>    at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
>    at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
>    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
>    at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
>    at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
>    at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
>    at System.Windows.Threading.DispatcherOperation.Invoke()
>    at System.Windows.Threading.Dispatcher.ProcessQueue()
>    at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
>    at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
>    at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
>    at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
>    at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
>    at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
>    at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
> 
>   This exception was originally thrown at this call stack:
>     [External Code]
>     NuGet.PackageManagement.UI.InfiniteScrollList.List_Loaded(object, System.Windows.RoutedEventArgs) in InfiniteScrollList.xaml.cs
>     [External Code]

**Comment by nkolev92** [comment]
Is this a regression @donnie-msft 

This sounds pretty bad. Not worried about the exp instance, but about regular VS.

**Comment by donnie-msft** [comment]
@nkolev92 yes, it certainly worked in 15.x, where I found the window did not try to reopen. (If I recall, it does try, but failures were ignored). 

I tested 16.7.4 and was able to repro. 
I recall similar HRResult errors and major refactoring occurring in the 16.5 timeframe. We had an Issue that was handed off to another team. I had to fix the auto-reopen of PMUI window https://github.com/NuGet/NuGet.Client/pull/3317. My guess is this would be a similar fix.
